dist/templates/ovnkube-cluster-manager.yaml.j2

```diff
@@ -0,0 +1,190 @@
+# ovnkube-cluster-manager
+# daemonset version 3
+# starts cluster manager daemon
+# it is run on the master(s)
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-cluster-manager
+  # namespace set up by install
+  namespace: ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This Deployment launches the ovn-kubernetes cluster manager networking component.
+spec:
+  progressDeadlineSeconds: 600
+  replicas: {{ ovn_cluster_manager_count | default(1|int) }}
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      name: ovnkube-cluster-manager
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: ovnkube-cluster-manager
+        component: network
+        type: infra
+        kubernetes.io/os: "linux"
+    spec:
+      priorityClassName: "system-cluster-critical"
+      # Requires fairly broad permissions - ability to read all services and network functions as well
+      # as all pods.
+      serviceAccountName: ovn
+      hostNetwork: true
```

Comment:
is there a need for cluster-manager to be host networked?

Reply:
I think so.

If a pod is not host networked, from where it will get the IP address from given that cluster-manager allocates and assigns subnets for each node and other non host network'd pods will get the IP from these subnets.